spotinst/oceancd_verification_template_metrics/consts.go

```diff
@@ -0,0 +1,99 @@
+package oceancd_verification_template_metrics
+
+import "github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+
+const (
+	Metrics          commons.FieldName = "metrics"
+	BaseLine         commons.FieldName = "baseline"
+	MaxRange         commons.FieldName = "max_range"
+	MinRange         commons.FieldName = "min_range"
+	BaseLineProvider commons.FieldName = "baseline_provider"
+	Threshold        commons.FieldName = "threshold"
+)
+
+const (
+	Datadog      commons.FieldName = "datadog"
+	Duration     commons.FieldName = "duration"
+	DatadogQuery commons.FieldName = "datadog_query"
+
+	NewRelic      commons.FieldName = "new_relic"
+	Profile       commons.FieldName = "profile"
+	NewRelicQuery commons.FieldName = "new_relic_query"
+
+	Prometheus      commons.FieldName = "prometheus"
+	PrometheusQuery commons.FieldName = "prometheus_query"
+)
+
+const (
+	ConsecutiveErrorLimit commons.FieldName = "consecutive_error_limit"
+	Count                 commons.FieldName = "count"
+	DryRun                commons.FieldName = "dry_run"
+	FailureCondition      commons.FieldName = "failure_condition"
+	FailureLimit          commons.FieldName = "failure_limit"
+	InitialDelay          commons.FieldName = "initial_delay"
+	Interval              commons.FieldName = "interval"
+	MetricsName           commons.FieldName = "metrics_name"
+	SuccessCondition      commons.FieldName = "success_condition"
+)
+
+const (
+	Provider   commons.FieldName = "template_provider"
+	Job        commons.FieldName = "job"
+	Jenkins    commons.FieldName = "jenkins"
+	CloudWatch commons.FieldName = "cloud_watch"
+)
+
+const (
+	CloudWatchDuration commons.FieldName = "cloud_watch_duration"
+	Expression         commons.FieldName = "expression"
```

Comment:
re-arrange Expression and MetricDataQueries

Reply:
re-arranged